test/e2e/engine/template.go

```diff
@@ -206,17 +206,19 @@ func (e *Engine) GetWindowsTestImages() (*WindowsTestImages, error) {
 		return nil, errors.New("Can't guess a Windows version without Windows nodes in the cluster")
 	}
 
-	if strings.Contains(e.ExpandedDefinition.Properties.WindowsProfile.GetWindowsSku(), "1809") || strings.Contains(e.ExpandedDefinition.Properties.WindowsProfile.GetWindowsSku(), "2019") {
+	windowsSku := e.ExpandedDefinition.Properties.WindowsProfile.GetWindowsSku()
+	switch {
+	case strings.Contains(windowsSku, "1809"):
```

Comment:
I don't think this is equivalent. Are we assuming the 1809 case will fall through to the 2019 case?

I think a preferable change would be to use a single func that can evaluate a string against a set of string values for equality.

Reply:
I added the `fallthrough` I forgot. I can pursue your suggested refactoring if that's preferable.

Reply:
+1 to jack's comment.

Reply:
can't we do  `case strings.Contains(windowsSku, "1809"), strings.Contains(windowsSku, "2019"):` if the two cases are the same result?

Reply:
We can, and I think that reads better than using `fallthrough` anyway.

Reply:
@jackfrancis @tariq1890 are you ok with the current changes?